docs/development/setup-advanced.md

```diff
@@ -53,112 +53,90 @@ on using the Zulip development
 environment](setup-vagrant.md#step-4-developing),
 ignoring the parts about `vagrant` (since you're not using it).
 
-## Installing directly on Windows 10 with WSL 2
+## Installing using Vagrant with VirtualBox on Windows 10
 
-Zulip's development environment is most easily set up on Windows using
-the Windows Subsystem for Linux ([WSL
-2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-about))
-installation method described here.
-
-1. Enable virtualization through your BIOS settings. This sequence
-   depends on your specific hardware and brand, but here are [some
-   basic instructions.][windows-bios-virtualization]
-
-1. [Install WSL 2](https://docs.microsoft.com/en-us/windows/wsl/setup/environment).
-
-1. Launch the Ubuntu shell as an administrator and run the following command:
-
-   ```bash
-   sudo apt update && sudo apt upgrade
-   ```
+:::{note}
+We recommend using [WSL 2 for Windows development](setup-vagrant.md#windows-10).
```

Comment:
Maybe add a few words, like "because it is easier to set up and provides a substantially better experience.".

Reply:
Done.